docs/en/analytics/segment/index.md

```diff
@@ -0,0 +1,38 @@
+# Segments
+A segment is a subset of your users. For example, of your entire set of users, one segment might be users who come from a same traffic source. Another segment might be users who trigger certain events, for example, made a purchase.
+
+## Access Segments
+To access Segments, follow below steps:
+
+1. Go to **Clickstream Analytics on AWS Console**, in the **Navigation Bar**, click on "**Analytics Studio**", a new tab will be opened in your browser.
+2. In the Analytics Studio page, click the **Segments** icon in the left navigation panel.
+
+## Create a segment
+1. Click on **Create Segment +** button
+2. Input **Segment name** and **Description**
+3. Specify a **Refresh method**
+    - Manual - Segment will be refreshed when user manually trigger refresh
+    - Automatically - Segment will be refreshed periodically. You can configure interval (e.g., Daily, Weekly, Monthly) and timing (e.g., 8AM). 
+4. Specify a **Expirations settings**. If you specified Automatically as refresh method, you will need to set an expiration date for a segment after which the segment will stop auto refresh.
```

Comment:
Automatically -> Automatic

Specify an Expiration settings.

Currently the expiration settings is not mandatory.